uint64 timestamp		# timestamp, UNIX epoch (GPS synced)
float64 lat			# target position (deg * 1e7)
float64 lon			# target position (deg * 1e7) 
float32 alt			# target position 
